specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -3160,6 +3167,39 @@
         }
       },
       "x-ms-discriminator-value": "IaasVMRestoreRequest"
+    },
+	"IdentityDetails": {
+      "type": "object",
+      "properties": {
+        "isSystemAssignedIdentity": {
+          "type": "boolean"
+        },
+        "managedIdentityArmUrl": {
+          "type": "string"
+        }
+      }
+    },
+    "IdentityInfo": {
+      "description": "Encapsulates Managed Identity related information",
+      "type": "object",
+      "properties": {
+        "objectType": {
+          "description": "Type of the specific object - used for deserializing",
+          "type": "string"
+        },
+        "isSystemAssignedIdentity": {
+          "description": "To differentiate if the managed identity is system assigned or user assigned",
+          "type": "boolean"
+        },
+        "managedIdentityArmUrl": {
+          "description": "Managed identity ARM URL\r\nOptional: Might not be required in the case of system assigned managed identity",
+          "type": "string"
+        },
+        "managedIdentityUrl": {
+          "description": "Managed Identity URL\r\nOptional: Will be required only in CRR flow, and not in non-CRR flow",
+          "type": "string"
+        }
+      }
     },
```

Comment:
Is this only change added in 2021-06-01 as well? Is there an example file show the usage of this?

Reply:
Next time please follow the ARM API Review Checklist that indicates how to make the very first commit for an efficient review (the second checkbox). Otherwise, it will be very hard to see what changed in the new api-version.

Reply:
+1 This is super important otherwise we'll need to review all the changes and will be adding comments to unmodified changes.